android/src/main/java/com/bugsnag/BugsnagReactNative.java

```diff
@@ -364,6 +369,7 @@ Severity parseSeverity(String value) {
         return output;
     }
 
+    @SuppressWarnings("unchecked") // ignore type erasure when casting Map
```

Comment:
Can this be scoped a bit smaller, e.g.:

```java
@SuppressWarnings("unchecked")
Map<String, Object> values = (Map<String, Object>) value;
```

Reply:
Reduced the suppression to local variable scope.